data/beaconrestapi/src/test/java/tech/pegasys/teku/beaconrestapi/handlers/v1/node/GetSyncingTest.java

```diff
@@ -70,9 +81,9 @@ void metadata_shouldHandle500() throws JsonProcessingException {
   void metadata_shouldHandle200() throws JsonProcessingException {
     final String data =
         getResponseStringFromMetadata(
-            handler, SC_OK, new GetSyncing.SyncStatusData(false, false, 10, 0));
+            handler, SC_OK, new GetSyncing.SyncStatusData(false, false, false, 10, 0));
     assertThat(data)
         .isEqualTo(
-            "{\"data\":{\"head_slot\":\"10\",\"sync_distance\":\"0\",\"is_syncing\":false,\"is_optimistic\":false}}");
+            "{\"data\":{\"head_slot\":\"10\",\"sync_distance\":\"0\",\"is_syncing\":false,\"is_optimistic\":false,\"el_offline\":false}}");
   }
```

Comment:
I'd like to see a test added that shows we can have the field not specified...

Reply:
Good catch! I added a test for the `isOptimistic` too...